### DISCUSSION TITLE
Welcome to xDrip Discussions!

### DISCUSSION THREAD

**Discussion by Osman5927** [NightscoutFoundation/xDrip#3535] [Q&A]
Hello everyone .please advice.
I can't receive messages about the night scout.
I went to Google Console, there was a 30 USD debt, and I made the payment. However, I still had 244 USD credit in the free credit section.
I cleared old data, restarted, synced, but it still didn't work.
Do I need to wait a bit or is there another problem?
Thank you very much

**Comment by Navid200** [comment]
It is not a good practice to cross post.
You asked the exact same question in facebook and I suggested what to do there.  

Decide which you want to use, facebook or here.

**Comment by Navid200** [comment]
The instructions for Google Cloud Nightscout are here: https://navid200.github.io/xDrip/docs/Nightscout/GoogleCloud.html

If you need help, you are welcome to ask here.  But, please don't cross post and please don't open an issue.

Post a screenshot of the status page as explained here: https://navid200.github.io/xDrip/docs/Nightscout/Status.html
